charters/wot-wg-charter-draft-2019.html

```diff
@@ -178,39 +186,369 @@ <h2>Scope</h2>
+        </p>
+        <p>
+          This work item would define an identity management lifecycle
+          that would seek, as much as possible, to mitigate the privacy
+          risks posed by explicit device identifiers while retaining
+          their functionality in database management and indexing.
+        </p>
+
+        <h3>Discovery</h3>
+        <p>During the previous charter a format for the storage of
+          the metadata required to access a Thing was defined: the
+          <a href="http://www.w3.org/TR/wot-thing-description">WoT Thing Description (TD)</a>.
+          However, no mechanism was defined for the distribution of TDs.
+          This is important since to preserve privacy, TDs should only
+          be distributed to parties with a permission to access the data.
+          However, we also wish to support flexible discovery of the services
+          and devices described by WoT TDs.
+        </p>
+        <p>
+          In this work item we will specify a privacy-preserving architecture
+          for Thing discovery.  
+          One possible way to do this is to distinguish
+          between an "introduction", which is a first-contact protocol, and 
+          "exploration", which retrieves detailed information about services.
+          Introductions can be public but should only provide the address
+          of an "exploration" service, which can be (for example) a web service
+          providing a searchable directory of TDs.  By making the 
+          addresses returned by introductions opaque and requiring authentication
+          for access to exploration services we can control who has access
+          to detailed information about specific devices.
+        </p>
+        <p>
+          This work item will require a coordinated effort with other standards
+          bodies such as IETF also working on discovery so that a common architecture
+          can be defined.  In addition, the architecture proposed by the Working Group
+          needs to support both peer-to-peer discovery and directory-based discovery.
+        </p>
+
+        <h3>TD Templates</h3>
+        <p>Current
+          <a href="http://www.w3.org/TR/wot-thing-description">WoT Thing Descriptions</a>
+          apply to specific Things, for example, particular devices.
+          However, there are often many devices manufactured with the same
+          interface and capabilities.  These devices differ from each other 
+          only in certain specific aspects, 
+          such as being installed in different locations
+          and having different identifiers.
+        </p>
+        <p>
+          The current TD specification includes the definition of a "TD Template"
+          which is a partial TD that omits information that is specific to an instance.
+          However, the current definition of TD Templates 
+          only allows definition of the interaction model without a protocol binding.  
```

Comment:
Templates provide device abstractions, that can be implemented by various devices in different ways. Not having a protocol binding is the core concept of the template. Various devices that implement the same template (i.e. data model) can be managed, simulated and handled in the same way. 
This is important for cloud platforms and gateways, that support various protocols, but manage the devices in a uniform way.

Reply:
There are places where it is useful to have a template without protocol bindings or security.   However, as a developer, I would need to know what protocol and security schemes I need to support to use a device, so I really just want a template that has everything EXCEPT (a) an id and (b) the "server" part of the URLs.   Anyway, let's discuss this point in the call.

Reply:
Although... profiles would constrain the protocols and security schemes that can be used, so that MIGHT be enough.

Reply:
I left this alone for now, we need to discuss more.